tests/acceptance/30_custom_promise_types/01_basic_module.cf

```diff
@@ -0,0 +1,68 @@
+######################################################
+#
+#  Basic test of custom promise types / promise modules
+#
+#####################################################
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  files:
+      "$(G.testfile)"
+      delete => init_delete;
+}
+
+body delete init_delete
+{
+      dirlinks => "delete";
+      rmdirs   => "true";
+}
+
+#######################################################
+
+promise agent example
+{
+    interpreter => "/bin/bash";
+    path => "$(this.promise_dirname)/example_module.sh";
+}
+
+bundle agent test
+{
```

Comment:
can you add a meta: "description" -> { "ticket" } string => "blah";

Reply:
Sure, added.